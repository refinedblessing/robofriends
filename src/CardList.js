import React from 'react';
import Card from './Card';

const CardList = ({ robots}) => robots.map((robot, i) => <Card key={i} {...robot}/>)

export default CardList;
