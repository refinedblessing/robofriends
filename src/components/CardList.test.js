import { shallow, mount, render } from 'enzyme'
import CardList from './CardList';
import React from 'react'

it ('enzyme works', () => {
  const mockRobots = [
    {
      id: 1,
      name: 'Jon Sonow',
      email: 'jj@gmail.com',
      username: 'Josn'
    }
  ]
  expect(shallow(<CardList robots={mockRobots}/>).length).toEqual(1)
  expect(shallow(<CardList robots={mockRobots}/>)).toMatchSnapshot();
})