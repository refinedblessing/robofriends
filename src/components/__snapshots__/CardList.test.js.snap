// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`enzyme works 1`] = `
<Card
  email="jj@gmail.com"
  id={1}
  key="0"
  name="Jon Sonow"
  username="Josn"
/>
`;
